lib/writer/block-write-stream.js

```diff
@@ -56,12 +58,14 @@ class BlockWriteStream extends stream.Writable {
     this.flags = options.flags
     this.mode = options.mode
     this.autoClose = options.autoClose
+    this.maxRetries = 5
```

Comment:
Does it make sense to store `maxRetries` on `this`? I can't imagine it ever changing, so perhaps a global `MAX_WRITE_RETRIES` const would be more appropriate?